samples/excel/42-range/range-cell-control.yaml

```
@@ -0,0 +1,163 @@
+order: 23
+id: excel-cell-controls
+name: Checkboxes
+description: This sample shows how to add and remove checkboxes from a table.
+host: EXCEL
+api_set:
+    ExcelApi: '1.18'
+script:
+    content: |-
+        document.getElementById("setup").addEventListener("click", () => tryCatch(setup));
+        document.getElementById("add-checkboxes").addEventListener("click", () => tryCatch(addCheckboxes));
+        document.getElementById("change-checkbox").addEventListener("click", () => tryCatch(changeCheckbox));
+        document.getElementById("remove-checkboxes").addEventListener("click", () => tryCatch(removeCheckboxes));
+        document.getElementById("remove-content").addEventListener("click", () => tryCatch(removeContent));
+
+        async function addCheckboxes() {
+          // Add checkboxes to the table.
+          await Excel.run(async (context) => {
+            const sheet = context.workbook.worksheets.getActiveWorksheet();
+
+            // Get the second column in the table.
+            const range = sheet.getRange("FruitTable").getColumn(1);
```

Consider not including the header in that column.
```suggestion
            const range = sheet.tables.getItem("FruitTable").columns.getItem("Analysis").getDataBodyRange();
```